stakingModel/test/GoodFundManager.test.js

```diff
@@ -183,6 +184,31 @@ contract(
       expect(tx.logs[0].event).to.be.equal("FundsTransferred");
     });
 
+    it("should create be able to stake dai", async () => {
+      const stakingMock = await SimpleDAIStakingMock.new(
+        dai.address,
+        cDAI.address,
+        goodFundManager.address,
+        0
+      );
+      await dai.mint(staker, web3.utils.toWei("100", "ether"));
+      await dai.approve(stakingMock.address, web3.utils.toWei("100", "ether"), {
+        from: staker
+      });
+      await stakingMock
+        .stakeDAI(web3.utils.toWei("100", "ether"), {
+          from: staker
+        })
+        .catch(console.log);
+        await cDAI.exchangeRateCurrent();
+        let stakingGDBalanceBefore = await goodDollar.balanceOf(stakingMock.address);
+        await goodFundManager.transferInterest(stakingMock.address);
+        let stakingGDBalanceAfter = await goodDollar.balanceOf(stakingMock.address);
+        expect(
+          stakingGDBalanceAfter.sub(stakingGDBalanceBefore).toString()
+        ).to.be.equal("190329"); // interest that the staking contract recieved
```

Comment:
yes but why 190329, show the calculation in comment

Reply:
db43a7633912fed7aa89a7ff580fbe5238d650b5